### DISCUSSION TITLE
how to install ragflow as a python package?

### DISCUSSION THREAD

**Discussion by adot08** [infiniflow/ragflow#3765] [Q&A]
I want to use the RAGFlow Python API. However, I can't find any relevant documentation for installing it as a Python package. Does anyone know how to install it?

**Comment by adot08** [comment]
Finally, I found a sdk in the project.
So we can:
```
cd sdk/python
pip install .
```
then
```
from ragflow_sdk import RAGFlow
```

**Comment by JinHai-CN** [comment]
https://pypi.org/project/ragflow-sdk/

pip install ragflow-sdk
